doc/api/api_changes/2016-12-09-AL_afm.rst

```diff
@@ -0,0 +1,4 @@
+``afm.get_fontconfig_fonts`` returns a list of paths and does not check for existence
+`````````````````````````````````````````````````````````````````````````````````````
+
+``afm.get_fontconfig_fonts`` used to return a set of paths encoded as a `{key: 1, ...}` dict, and checked for the existence of the paths.  It now returns a list and dropped the existence check, as the same check is performed by the caller (``afm.findSystemFonts``) as well.
```

Comment:
Pretty sure the backticks are the wrong way here, in that single ticks are references and double are code formatting (or at least the dict one needs to be doubled if the other references are not wanted.)